docs/dev/table/catalog.md

```diff
@@ -161,8 +161,8 @@ Currently `HiveCatalog` supports most Flink data types with the following mappin
 | VARCHAR(p)    |  varchar(p)** |
```

Comment:
nit: make all Hive data types upper case, this is also how the Hive documentation explains them: https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types#LanguageManualTypes-NumericTypes